menu_trans.vim.template

```diff
@@ -0,0 +1,1255 @@
+" Menu Translations:    Simplified Chinese
+" Maintainer:           Shun Bai <baishunde@gmail.com>
+" Previous Maintainer:  Yuheng Xie <elephant@linux.net.cn>
+" Last Change:          2022 Feb 19
+" Original translations
+
```

Comment:
Add a few lines of comments to describe that the generated translation files are creating using your tool. Provide link to your GitHub repository.